examples/job_posting/use_planner.py

```diff
@@ -18,6 +18,9 @@
 
 # Load environment variables
 load_dotenv()
+AGENT_RESEARCHER_ROLE = """An expert in gathering information about a job. Goal is to analyze the company website and provided description to extract insights on culture, values, and specific needs."""  # noqa: E501
```

Comment:
This is a simple single line string, so please use `"` instead of `"""` with porper formatting and remove `# noqa: E501`. Please do all such cases